src/installer/dbinstall_mysql.sql

```diff
@@ -36,6 +36,7 @@ INSERT INTO `DBPREFIXconfig` (`identifier`, `type`, `value`, `user_editable`) VA
 ('adminstatus_offlinehiddenbydefault', 'BOOL', 'false', 1),
 ('imprint_enabled', 'BOOL', 'false', 1),
 ('imprint_url', 'STRING', 'imprint.php', 1),
+('assigner_cooldown_period', 'INT', '0', 1),
```

Comment:
`assigner_cooldown_seconds` would be a better name (it makes it obvious that this value contains seconds)

Reply:
The name has no seconds because the rest of the stuff in the database doesn't contain it.